src/core/support/cpu_windows.c

```diff
@@ -34,19 +34,17 @@
 #include <grpc/support/port_platform.h>
 
 #ifdef GPR_WIN32
-
+#include <Windows.h>
```

Comment:
Please use <windows.h> in lowercase instead. If cross compiling from Linux, that would fail otherwise.